lib/module.js

```diff
@@ -285,12 +276,11 @@ Module._resolveLookupPaths = function(request, parent) {
     return [request, []];
   }
 
-  var reqLen = request.length;
   // Check for relative path
-  if (reqLen < 2 ||
-      request.charCodeAt(0) !== 46/*.*/ ||
-      (request.charCodeAt(1) !== 46/*.*/ &&
-       request.charCodeAt(1) !== 47/*/*/)) {
+  if (request.charCodeAt(0) !== 46/*.*/ || (
+      request.charCodeAt(1) !== 46/*.*/ &&
+      request.charCodeAt(1) !== 47/*/*/ &&
+      request.length > 1)) {
```

Comment:
@mscdex does this change look right to you? The condition was `start !== '.' && start !== './' && start !== '..'` before you refactored it to use `charCodeAt`. Tests are passing, but I'd like to get a second look on that one.